.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -3,6 +3,7 @@ on:
   schedule:
     # Run this workflow "At 20:00 UTC on Sunday and Saturday"
     - cron: '0 20 * * 0,6'
+  pull_request:
```

Comment:
It's working for show the CI, it will be deleted once i get the LTGM

Reply:
LGTM